### ISSUE TITLE
[FE] 1010d/10-7959c Merge Staging Review finding: Error state and instructions are not clear

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#118214]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** IVC Forms
**Product:** 10-10D CHAMPVA Benefits and 10-7959c OHI
**Feature:** Merged form solution

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Add multiple Medicare plans/recipients > On Review & Submit > Delete the last added Plans under "Medicare information"  > Submit form. 

The error messages do not explicitly indicate what the issue is or how to correct it. 

Note: There are several error states that can be triggered in various ways of interacting with Review & Submit. 

Another path:
- From the above error state, delete all plans under Medicate information and Health insurance information. This clears the error
- Click Add another plan under Medicare information
- Fill out all required details through the flow
- Back at Review and Submit, note error message (second screenshot)

Note:
- There was another scenario where there was an error message that read "Please return to the following 0 parts of the form" without any items listed. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="693" height="755" alt="medicare unhelpful error" src="https://github.com/user-attachments/assets/50c8fba2-d7a3-4dfd-944e-868321ba6352" />
<img width="517" height="904" alt="health insurance info unhelpful error" src="https://github.com/user-attachments/assets/1c7893a9-1d7f-4892-81c5-3c0380eca0ed" />

</details>

## Recommended action

Ensure the error explicitly indicates what the user needs to do to correct the entered data, and where in the form that needs to occur. 

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 3.3.3 Error Suggestion - Level AA](https://www.w3.org/TR/WCAG22/#error-suggestion)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

## Acceptance Criteria
- [ ] Add a check for invalid or missing required fields and show a user-friendly warning to the user
- [ ] Add/Update tests

**Comment by longmd** [comment]
Thanks @rhasselle-oddball for doing the spike on this. I was unaware of the opt-in setting and thought it was a standard freature. I did some looking at it's adoption rate and it looks like only 13 products are using this setting and of those, 6 of them are using it only for lower ENVs and not for prod.

After review, we are going to opt out for the time being. Depending on whether y'all choose to pick up the work described in your ticket and push that forward, we may revisit our opt status. We're always happy to collaborate on any testing or use of that in the future.

**Comment by jasonday** [comment]
@longmd or @tbaker1026 - can y'all clarify the decision point, and the status? Has this been fixed or deemed won't fix?

**Comment by jasonday** [comment]
Thanks @tbaker1026  - ok, so 10-10 has opted out of the reviewErrors component, meaning that for their product, the finding is no longer valid. I'm fine with this ticket remaining closed. Thanks!
